Basic FE project template/Gruntfile.js

```diff
@@ -6,6 +6,8 @@ module.exports = function (grunt) {
           "ui-src/js/test.js",
           "ui-src/js/utilities.js",
           "ui-src/js/animatedCounter.js",
+          "ui-src/js/hoverBar.js",
```

Comment:
why did you create 2 files for the menu? why is one called "slidebar" when you used "sidebar" classes? You can make only 1 js file menu.js